src/mount/fuse/mount_config.cc

```diff
@@ -161,13 +163,16 @@ void usage(const char *progname) {
 "Write related options:\n"
 "    -o sfschunkserverwriteto=MSEC  set chunkserver response timeout during "
 				"write operation in milliseconds (default: %u)\n"
+"    -o sfschunkserverwavewriteto=MSEC  set timeout for executing each wave "
+				"of a write operation in milliseconds (default: %u)\n"
 "    -o sfswritecachesize=N      define size of write cache in MiB (default: %u)\n"
 "    -o sfscacheperinodepercentage=P  define what part of the write cache non "
 				"occupied by other inodes can a single inode "
 				"occupy (in %%, default: %u)\n"
 "    -o sfswriteworkers=N        define number of write workers (default: %u)\n"
 "    -o sfswritewindowsize=N     define write window size (in blocks) for "
 				"each chunk (default: %u)\n"
+"    -o sfsuseoldwritealgorithm  use legacy write algorithm.\n"
```

Comment:
Consider adding the default value here.

Reply:
Overall, change the option to be explicit, and apply related suggestions.